src/shared/components/palette/address/write-address-field.component.ts

```diff
@@ -70,6 +74,16 @@ export class WriteAddressFieldComponent extends AbstractFieldWriteComponent impl
           );
         });
       this.addressList.setValue(undefined);
+      this.refocusElement();
+    }
+  }
+
+  refocusElement(): void {
+    // The focus element directive focuses on the element that uses the focus directive. In this case it will focus
+    // on address list dropdown. This requires explicit calling as the directive only focuses the element the very
```

Comment:
Perhaps a better place for this would be to the FocusElementDirective itself?

Reply:
👍